modules/core/02-client/client/cli/tx.go

```diff
@@ -79,7 +79,6 @@ func NewCreateClientCmd() *cobra.Command {
 	}
 
 	flags.AddTxFlagsToCmd(cmd)
-
```

Comment:
[nit] unnecessary changes to this file

Reply:
Other commands in the same file had that statement next to the return, so I removed that empty line for consistency. I can revert it, if you want; have no strong preference...

Reply:
ah okay, no it's not a big deal at all!

Reply:
I actually kinda like the spaces, a little confusing why we format differently between query.go and tx.go